upp/classes/resampling_config.py

```diff
@@ -11,6 +11,7 @@ class ResamplingConfig:
     target: str
     sampling_fraction: float = 1.0
     method: str | None = None
+    upscale_pdf: int = 1
```

Comment:
I'd consider making this `None` or `0` by default so we can later conviently check

```python
if self.upscale_pdf:
    <do stuff>
```